packages/kbn-pm/src/commands/bootstrap.ts

```diff
@@ -33,7 +33,11 @@ export const BootstrapCommand: ICommand = {
     const batchedProjects = topologicallyBatchProjects(projects, projectGraph);
 
     const frozenLockfile = options['frozen-lockfile'] === true;
-    const extraArgs = frozenLockfile ? ['--frozen-lockfile'] : [];
+    const offline = options.offline === true;
+    const extraArgs = [
+      frozenLockfile ? '--frozen-lockfile' : '',
+      offline ? '--offline' : '',
```

Comment:
Does `--offline` require that we update the cache repo with new dependencies before they will pass CI? Should we perhaps use `--prefer-offline`?

Reply:
it does, good call.  do we use prefer-offline here, and keep it at offline for builds?  the idea being that we do want to fail somewhere if we're not cached?  

the more it i think about it -offline sounds like maintenance nightmare.  what if we based it on the release flag?
`--release ? '--offline' : '--prefer-offline'`